zap/src/main/java/org/zaproxy/zap/extension/stdmenus/PopupContextTreeMenu.java

```diff
@@ -62,8 +110,24 @@ public boolean isEnabledForContext(int contextId) {
         return true;
     }
 
+    /**
+     * Returns whether or not the menu should be enabled when multiple contexts are currently
+     * selected, by default True.
+     *
+     * @return Whether or not the menu should be enabled on multiple contexts
+     * @since TODO Add version
+     */
+    public boolean isEnabledForMultipleContexts() {
```

Comment:
This could be protected (more an internal detail than for external consumption.)